example.config.yaml

```diff
@@ -31,3 +32,7 @@ commands:
       - "{{time}} left to perfect your masterpiece!"
       - "{{time}} left. PANIC! \:D/"
       - "Dreams will be crushed in {{time}} and counting."
+  random:
+    pending: "{{event.title}} has not started yet"
+    empty: "No entries for {{event.title}}"
+    entry: "Random entry for {{event.title|safe}}: '{{entry.title|safe}}' by{% set sep = joiner() %}{% for user in entry.users %}{{sep()}} {{user.title|safe}}{% endfor %}  https://alakajam.com/{{entry.event_name}}/{{entry.id}}"
```

Comment:
I wouldn't worry too much about this being hard-coded to alakajam.com&mdash;we're linking to it above, and this is just the example config anyway.

If you're keen to tidy up the template one option is to use some sort of [block notation](http://www.yaml.org/spec/1.2/spec.html#id2760844), combined with nunjucks' whitespace control (`{%-` and `-%}`). For example, this is a bit more readable:

     entry: >
      Random entry for {{event.title|safe}}: '{{entry.title|safe}}' by
      {%- set sep = joiner() -%}
      {%- for user in entry.users -%}
        {{sep()}} {{user.title|safe}}
      {%- endfor %}:
      https://alakajam.com/{{entry.event_name}}/{{entry.id}}

With some dummy data this renders to `'Random entry for EventTitle: \'EntryTitle\' by UserA, UserB: https://alakajam.com//\n'`.

Reply:
Indeed, the template turned out a bit ugly. While I was reading the documentation for the whitespace control, I noticed that there is a cleaner way to join: using a filter. Thanks for the comment!